app/presenters/tree_builder_alert_profile_obj.rb

```diff
@@ -0,0 +1,72 @@
+class TreeBuilderAlertProfileObj < TreeBuilder
+  def initialize(name, type, sandbox, build = true, assign = nil)
+    @assign = {
```

Comment:
You are absolutely right, it may be problem to the future with refactoring, i did not realize that.